trieste/models/model_interfaces.py

```diff
@@ -98,6 +98,21 @@ def get_observation_noise(self) -> TensorType:
         """
         raise NotImplementedError(f"Model {self!r} does not provide scalar observation noise")
 
+    def covariance_between_points(
```

Comment:
Why has this method been added to the interface? It doesn't appear to be used in the PR. If it is not needed here, could the change be deferred until it is needed?

Reply:
My mistake, Crept in from another branch